packages/target-tauri/src-tauri/src/lib.rs

```diff
@@ -86,6 +89,11 @@ fn get_current_logfile(state: tauri::State<AppState>) -> String {
 pub fn run() {
     let startup_timestamp = SystemTime::now();
 
+    #[cfg(desktop)]
+    let run_config = cli::parse_cli_options();
```

Comment:
Do we want this in production? This pulls in `chrono` and `clap` dependencies. Did you check the build size change?

Reply:
size increase is not even a mb, maybe ~170kb